clusterloader2/testing/load/config.yaml

```diff
@@ -229,6 +236,7 @@ steps:
       objectTemplatePath: daemonset.yaml
       templateFillMap:
         Image: k8s.gcr.io/pause:3.0
+  {{if not $IS_SMALL_CLUSTER}}
```

Comment:
Is this if block needed here? I believe the `$bigDeploymentsPerNamespace` will be 0 if `.Nodes` < 100. In that case no object is going to be created

Reply:
Will try again, I may have mixed this up with the error coming from a negative `$bigDeploymentsPerNamespace`

Reply:
Big deployment objects are still being created even though `bigDeploymentsPerNamespace` is 0. Seems like ReplicasMin and ReplicasMin have the final say. So maybe targeting the `BIG_GROUP` size variable would be better as it will affect jobs created at the same time.

![image](https://user-images.githubusercontent.com/13202538/106619505-a0694a80-65ab-11eb-9b1d-78c10952587e.png)

Reply:
Nah, we're missing something. If the `replicasPerNamespace` is set to 0 then no new object should be created - https://github.com/kubernetes/perf-tests/blob/8c85b1d608511e76d8c8180b3bc6163b736e20ec/clusterloader2/pkg/test/simple_test_executor.go#L261

The only explanation is that the $bigDeploymentsPerNamespace` is not set to 0.  Isn't it because in line 67 we're doing:
```
{{$bigDeploymentsPerNamespace := MaxInt 1 (SubtractInt $bigDeploymentsPerNamespace 1)}}
```
?

Reply:
Yes! This is it, addressed in my latest commit https://github.com/kubernetes/perf-tests/pull/1667/commits/99c29b498be48a07791d7b92e500bba406af45f9